app/views/finance/lead_providers/show.html.erb

```diff
@@ -0,0 +1,98 @@
+<% content_for :title, "Lead Provider Payment Breakdown" %>
+<%
+  def band_index_to_heading(number)
+    ("A".."Z").to_a[number]
+  end
+%>
+
+<h1 class="govuk-heading-l"><%= @ecf_lead_provider.name %></h1>
+
+<table class="govuk-table">
+  <caption class="govuk-table__caption govuk-table__caption--l">ECF Payment breakdown</caption>
+  <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header govuk-!-width-one-half">Milestone</th>
+        <td class="govuk-table__cell" data-test="lead-provider-name">Started</td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header">Recruitment target</th>
+        <td class="govuk-table__cell" data-test="lead-provider-name"><%= @ecf_lead_provider.call_off_contract[:recruitment_target] %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header">Current ECTs</th>
+        <td class="govuk-table__cell" data-test="lead-provider-name"><%= @total_ect %></td>
```

Comment:
Generally, I'm not sure we have enough test coverage on this patch, I'm not currently confident that tests would catch breakages if numbers went missing. (I got the PR out anyway so we could get moving and make a more shared decision). Making a comment on a line so we can use the "resolve" feature of github.

Reply:
Percy snapshots would catch those numbers missing (I hope). If you want to make doubly sure, you can check the `assigns` of `total_ect` variable in rspec - similar to
```
  describe "GET /finance/lead-providers" do
    it "renders the index template" do
      get "/finance/lead-providers"

      expect(response).to render_template("finance/lead_providers/index")
      expect(assigns(:lead_providers)).to eq(lead_providers)
    end
  end
```